static/include/app/apis/generated/app.md

```diff
@@ -2587,7 +2622,7 @@ User-defined metadata is billed as data.
 
 **Parameters:**
 
-- `robot_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the robot with which to associate the user-defined metadata. You can obtain your robot ID from your machine's page.
+- `robot_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the robot with which to associate the user-defined metadata. You can obtain your robot ID from the machine page.
```

Comment:
This seems less helpful

Reply:
This is generated documentation from the SDK doc. I'll file a PR to improve this wording but I think it's out of scope and would hold this PR up for a while while we fix it upstream.